contrib/intel/jenkins/run.py

```diff
@@ -60,6 +60,17 @@ def shmemtest(core, hosts, mode, util=None):
               .format(runshmemtest.testname))
     print("----------------------------------------------------------------------------------------\n")
 
+def zetest(core, hosts, mode, util=None):
```

Comment:
I think a  typo here - ze_test  instead of zetest?

Reply:
yes, changed it.